modules/apps/shared-dependencies/shared-dependencies-jericho-html/bnd.bnd

```diff
@@ -0,0 +1,4 @@
+Bundle-Name: Liferay Shared Dependencies jericho-html
```

Comment:
@jeffwu0724 Please rename the bundle name to : Liferay Shared Dependencies **Jericho Html**